### PR TITLE
combine #[cfg(...)] for templates into macro

### PR DESCRIPTION
**Pull request by harrysarson**
hopefully makes the logic easier to read and maintain

### CODE REVIEW

**Comment by mpizenberg** [comment]
That issue seems relevant to the compilation failure here: https://github.com/rust-lang/rust/issues/53749

File: src/utils.rs [hunk 1]
```diff
@@ -3,6 +3,22 @@
...
+    ($name:expr) => {
+        include_str!(concat!("../templates/", $name))
+    };
+}
+
+#[macro_export]
+#[cfg(windows)]
+macro_rules! include_template {
+    ($name:expr) => {
+        include_str!(concat!("..\\templates\\", name))
```
Comment (mpizenberg) [inline src/utils.rs:18]:
```suggestion
        include_str!(concat!("..\\templates\\", $name))
```

**Comment by mpizenberg** [comment]
Sorry for the multiple notifications and thanks for the refactor! That's indeed cleaner and easier to follow. If you don't have things to add, that's ready for merge.

**Comment by harrysarson** [comment]
Is it a problem that CI fails?

**Comment by mpizenberg** [comment]
MacOS tests are failing when CI loads the cache, I don't know why. If you look at the first CI results on you commit, you'll see that MacOS tests were passing. It's a bit annoying and if you know why, fixing it would be awesome but we can fix it in another PR

**Comment by harrysarson** [comment]
Let's merge then 👍

**Pull request merged**